### PR TITLE
Home screen

### PR DESCRIPTION
**Pull request by iamsammak**
#153 rearrange home screen.

Pushing this so that we can all look at it and give opinions.
But I still want to fiddle with the layout, size and colors?

Screen Shot 2019-03-13 at 4 27 12 PM

### CODE REVIEW

File: frontend/components/main.jsx [hunk 1]
```diff
@@ -3,106 +3,119 @@ import { Link } from 'react-router-dom';
 import styled from 'styled-components';
 
 import { PEOPLE } from '../helpers';
-import { xsSpacing } from '../constants';
+import { sidecar, tamarind, juniper, dustStorm, whiteLilac } from '../constants';
```
Comment (ihurrahi) [inline frontend/components/main.jsx]: doesn't look like these are all used?

File: frontend/components/main.jsx [hunk 2]
```diff
@@ -3,106 +3,119 @@ import { Link } from 'react-router-dom';
...
     return (
       <div>
         <WelcomeHeader {...this.props} />
         <Table>
           {
-            items.map((item, index) => {
-              const rotation = index * (360 / items.length);
+            firstRow.map((item) => {
+              return (
+                <StyleLink to={item.to} key={item.key}>
```
Comment (ihurrahi) [inline frontend/components/main.jsx]: these all look the same - move to a `renderPerson` method? makes the jsx really simple:
```
<Table>
  {firstRow.map(this.renderPerson)}
</Table>
```

File: frontend/components/main.jsx [hunk 3]
```diff
@@ -3,106 +3,119 @@ import { Link } from 'react-router-dom';
...
+  justify-content: center;
   width: ${tableSize}px;
-  height: ${tableSize}px;
+  padding: 5px 0;
   margin: auto;
 `;
 
-const PersonContainer = styled.div`
-  padding: ${xsSpacing}px;
+const MiddleTable = styled(Table)`
```
Comment (ihurrahi) [inline frontend/components/main.jsx]: no need for a separate MiddleTable defintion

File: frontend/components/main.jsx [hunk 4]
```diff
@@ -3,106 +3,119 @@ import { Link } from 'react-router-dom';
...
 `;
+// justify-content: space-evenly;
```
Comment (ihurrahi) [inline frontend/components/main.jsx]: remove comment (similarly below)

File: frontend/components/main.jsx [hunk 5]
```diff
@@ -3,106 +3,119 @@ import { Link } from 'react-router-dom';
 import styled from 'styled-components';
 
 import { PEOPLE } from '../helpers';
-import { xsSpacing } from '../constants';
+import { sidecar, tamarind, juniper, dustStorm, whiteLilac } from '../constants';
 import WelcomeHeader from './welcomeHeader';
 
 const tableSize = 350;
```
Comment (ihurrahi) [inline frontend/components/main.jsx]: don't think we need this anymore, i think it was used to help with the calculations for where to place things

File: frontend/components/main.jsx (hunk 1 above)
Comment (iamsammak) [reply frontend/components/main.jsx]: Yeah I was playing around with the colors, I'll remove the ones I don't end up using

File: frontend/components/main.jsx (hunk 5 above)
Comment (iamsammak) [reply frontend/components/main.jsx]: yup I'll delete

File: frontend/components/main.jsx (hunk 3 above)
Comment (iamsammak) [reply frontend/components/main.jsx]: Yeah I think I justify-content: center looks the best for the middle section so I'll delete this along with all my comment changes that I was playing around with

File: frontend/components/main.jsx (hunk 2 above)
Comment (iamsammak) [reply frontend/components/main.jsx]: I like this idea. I'll implement a renderPerson method

**Pull request merged**